bootstrap/src/lib/metrics/index.ts

```diff
@@ -0,0 +1,39 @@
+import * as client from 'prom-client'
+
+const collectDefaultMetrics = client.collectDefaultMetrics
+const Registry = client.Registry
+export const register = new Registry()
+
+collectDefaultMetrics({ register })
+
+export const httpRequestsTotal = new client.Counter({
+  name: 'http_requests_total',
+  help: 'The number of http requests this external adapter has serviced for its entire uptime',
+  labelNames: ['method', 'statusCode', 'apiKey', 'retry'] as const,
```

Comment:
I would merge `cacheWarmerRequests`, `httpRequestsCacheHits`, `httpRequestsDataProviderHits` into this metric as labels, like the following:
```ts
labelNames: ['method', 'statusCode', 'apiKey', 'retry', 'isCacheHit', 'isWarmerRequest', 'isDataProviderHit'] as const,
```

Reply:
This way, you can set those additional labels as boolean values

Reply:
Now that i think about it, we dont even need `isDataProviderHit` since we can infer that from `isCacheHit`

Reply:
Perfect, didn't know that